README.md

```diff
@@ -96,3 +96,33 @@ First, make sure you have run the "Create some data" step above.
 1. Enter the web container: `docker-compose exec web bash`
 2. Build the indicies: `./manage.py esreindex` (You may need to pass the `--delete` flag)
 3. Precompile the nunjucks templates: `./manage.py nunjucks_precompile`
+
+## Development in vscode (beta)
```

Comment:
I don't think that we should include any editor specific instructions in the main README file. I would be fine with that if we could provide instructions for more editors. Also this workflow may not be for anyone working with vscode. Let's move this section to a readme in the vscode dir.

Reply:
Makes sense :+1: